### ISSUE TITLE
JDK Versions section could be partly adjusted to refer to Java language levels rather than JDK versions

### ISSUE THREAD

**Issue by SonicGDX** [libgdx/libgdx.github.io#216]
The information in the table in the Versions section of https://libgdx.com/wiki/articles/java-development-kit-selection seems accurate to my knowledge (after it was updated in https://github.com/libgdx/libgdx.github.io/pull/198), except that by versions it is referring to the JDK version when I think language level is more appropriate (although maybe not for stuff about DCEVM and the notes about oracle licensing and which JDKs are LTS)

- According to [the readme](https://github.com/libgdx/gdx-liftoff/blob/master/README.md) from gdx-liftoff, all stable JDK releases from 8 to 22 work (I'd keep the part about LTS releases being recommended though), but JDK 17 or higher is recommended because the newer android tools versions used require a JDK of 17 or higher.

**Comment by Frosty-J** [comment]
I agree... needing to use multiple JDKs is atypical. In most cases, `source`/`targetCompatibility` takes care of this.

Android versions don't fit nicely into buckets and I don't know what that information (JDK 7 for Android 6, JDK 8 for Android 7...) is based on. For all intents and purposes, Android presently requires JDK 17 for the reason you say and is happy with any language level up to and including 17, no matter how low your `minSdk`. And what is with the empty ARM column?

**Comment by crykn** [comment]
This page has been bugging me since it has been added... IMO it should also do a better job at explaining the difference between JRE, JDK, Language Level, `sourceCompatibility`, `targetCompatibility` etc.
